Packs/CommonScripts/ReleaseNotes/1_11_14.md

```diff
@@ -1,4 +1,4 @@
 
 #### Scripts
 ##### GetTime
-- Documentation and metadata improvements.
\ No newline at end of file
+- Documentation and metadata improvements.
```

Comment:
This file shouldn't be here, please revert it from master

Reply:
Done.